var/spack/repos/builtin/packages/py-alphafold/package.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import glob
+
+
+class PyAlphafold(PythonPackage, CudaPackage):
+    """AlphaFold is an AI system developed by DeepMind that predicts a
+    protein's 3D structure from its amino acid sequence. It regularly
+    achieves accuracy competitive with experiment."""
+
+    homepage = 'https://github.com/deepmind/alphafold'
+    url = 'https://github.com/deepmind/alphafold/archive/refs/tags/v2.1.1.tar.gz'
+
+    version('2.1.1', sha256='1adb6e213ba9ac321fc1acb1c563ba9b4fc054c1cebe1191bc0e2aaa671dadf7')
+
+    depends_on('python@3.7:', type=('build', 'run'))
+    depends_on('py-absl-py@0.13.0:', type=('build', 'run'))
+    depends_on('py-biopython@1.79:', type=('build', 'run'))
+    depends_on('py-chex@0.0.7:', type=('build', 'run'))
+    depends_on('py-dm-haiku@0.0.4:', type=('build', 'run'))
+    depends_on('py-dm-tree@0.1.6:', type=('build', 'run'))
+    depends_on('py-immutabledict@2.0.0:', type=('build', 'run'))
+    depends_on('py-jax@0.2.14:', type=('build', 'run'))
+    for arch in CudaPackage.cuda_arch_values:
+        depends_on('py-jax@0.2.14:+cuda cuda_arch={0}'.format(arch),
+                   type=('build', 'run'),
+                   when='cuda_arch={0}'.format(arch))
+    depends_on('py-ml-collections@0.1.0:', type=('build', 'run'))
+    depends_on('py-numpy@1.19.5:', type=('build', 'run'))
+    depends_on('py-pandas@1.3.4:', type=('build', 'run'))
+    depends_on('py-scipy@1.7.0:', type=('build', 'run'))
+    depends_on('py-pdbfixer@1.7', type=('build', 'run'))
+    depends_on('py-tensorflow@2.5:2.6', type=('build', 'run'))
```

Comment:
Where do you see these dependency versions? All I see is:
```python
    install_requires=[                                                                   
        'absl-py',                                                                       
        'biopython',                                                                     
        'chex',                                                                          
        'dm-haiku',                                                                      
        'dm-tree',                                                                       
        'docker',                                                                        
        'immutabledict',                                                                 
        'jax',                                                                           
        'ml-collections',                                                                
        'numpy',                                                                         
        'pandas',                                                                        
        'scipy',                                                                         
        'tensorflow-cpu',                                                                
    ],         
```
Some of these are missing btw.

Reply:
used requirements.txt as hints - docker is missing as I didn't install the docker scripts.